README.md

```diff
@@ -20,6 +20,22 @@ curl http://localhost:8080/tasks/populate-dev-data
 
 See [CONTRIBUTING.md](/CONTRIBUTING.md) for more information on local development.
 
+## Running the tests
+
+### Setup
+
+You'll need to make sure that you have Python 3 installed. It is recommended that you setup a [virtualenv](https://virtualenv.pypa.io/en/stable/). When you have activated your `virtualenv`
+run the following command
+```sh
+  pip install -r requirements
+```
+
+### Running
+
+Copy the file `run/running.example.ini` and edit the fields to the correct locations for items on your machine.
+If you do not do this you will receive a `configparser.NoSectionError: No section: 'default'` message in your
+console.
```

Comment:
I think instead of having this sentence, there should be a warning in `run/run.py` if the file is not detected.